### PR TITLE
feat(loginPage)User should  change their passwords every x amount of time [187364810]

### PR DESCRIPTION
**Pull request by shebz2023**
**what does this PR do?**

this Pr adds an interface that tells users with expired password to change their password 
it achieves this by navigating users with expired passwords to the change page and there they can find a link to change their password ,all this is for their security.

**what are relevant pivotal tracker id?**
187364810

1➡️2

Screenshot 2024-06-28 202440_Combine

### CODE REVIEW

**Review by martinemahirwe — DISMISSED** [review]
Good work @shebz2023

**Review by Sagastein — DISMISSED** [review]
LGTM 🆗

File: src/pages/authpages/RequestPwdChangePage.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import leftImage from '../../assets/ipad_pro__ea93uc0cnvsm_large_2x.jpg';
+import { RequestPwdChangeForm } from '../../components';
+export const RequestPwdChange = () => {
+  return (
+    <div className="flex bg-white">
+      <div 
+        className="hidden lg:block lg:w-1/2 bg-cover bg-center h-screen" role='image'
+        style={{ backgroundImage: `url(${leftImage})` }}
+      ></div>
```
Comment (SergeInGithub) [inline src/pages/authpages/RequestPwdChangePage.tsx]:
```suggestion
      <div 
        className="hidden lg:block lg:w-1/2 bg-cover bg-center h-screen" role='image'
        style={{ backgroundImage: `url(${leftImage})` }}
      />
```

**Review by SergeInGithub — CHANGES_REQUESTED** [review]
Address the minor change below and you're good

**Review by SergeInGithub — APPROVED** [review]
LGTM

**Pull request merged**